### PR TITLE
ID#11428 - Oracle Integration 3 - Understand Rest Adapter

### PR DESCRIPTION
**Pull request by kkatta11**
# 11428

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [YES ] All filenames are lower case (including folders, images, files, etc.)
- [YES ] Filenames are descriptive
- [YES ] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [NO ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [YES ] Image references in markdown contain an alternate text

### CODE REVIEW

File: oic-gen3/core-competencies/get-started-oic/create-flow/create-flow.md [hunk 1]
```diff
@@ -195,11 +195,11 @@ When we added the ADW invoke to the integration, a map icon was automatically ad
 
 ## Task 6: Activate the integration
 
-1. On the *Integrations* page, click on the **Activate** icon.
+1. On the *Integrations* page, however over the integration you created, and click the **Activate** icon
```
Comment (denisemyrick) [inline oic-gen3/core-competencies/get-started-oic/create-flow/create-flow.md:198]: I think however is supposed to be hover

File: oic-gen3/core-competencies/understand-rest-adapter/adb-provision/adb-provision.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+
+3. Specify basic information for the autonomous database:
+
+    | **Field**  | **Value** | Note |
+    |---|---|---|
+    | Compartment | (Defaut) | |
+    | Display Name | `OICDemo` | Enter a memorable name for the database for display purposes |
+    | Database Name | `OICDEMO` | Use letters and numbers only, starting with a letter. Maximum length is 14 characters. |
+    |
```
Comment (denisemyrick) [inline oic-gen3/core-competencies/understand-rest-adapter/adb-provision/adb-provision.md:75]: recommend deleting this |

File: oic-gen3/core-competencies/understand-rest-adapter/create-flow/create-flow.md [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+    - Click **&gt; (Next step)**.
+
+4. From the *Request Parameters* page, in the *Specify Query Parameter section*, click **Add**. A new row appears.
+    - Enter **customer_id** in the *Name* column and select **string** from the *Date Type* column
+    - Click **&gt; (Next step)**.
+    ![Query Parameter Page](images/rest-adapter-query-parameter-page.png)
+
+5. On the **Response** Page
+    - Select the *response payload format* to **JSON Sample**
+    - Click the **&lt&lt&ltinline&gt&gt&gt** link.
```
Comment (denisemyrick) [inline oic-gen3/core-competencies/understand-rest-adapter/create-flow/create-flow.md:79]: Is this right?

Comment (kkatta11) [reply oic-gen3/core-competencies/understand-rest-adapter/create-flow/create-flow.md:79]: Yes it is right the UX has <<inline>> so we need to espace

File: oic-gen3/core-competencies/understand-rest-adapter/create-flow/create-flow.md [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+1.  Delete the Map activity created after the Trigger activity and **Save** the Integration Flow
+
+2. Hover over the outgoing arrow for the *getCustomer* activity (after first activity) and Click the **+** sign in the integration canvas.
+Search for the *REST Inventory OAS* connection which was created in the previous Lab and click on it. This invokes the REST  Adapter Configuration Wizard.
+
+3. On the **Basic Info** page,
+    - for the *What do you want to call your endpoint?* element, enter **getCustomer**
+    - Click **&gt; (Next step)**.
+
+3. On the *Operations* page,
```
[the remark this answers is not in the document]
Comment (kkatta11) [reply oic-gen3/core-competencies/understand-rest-adapter/create-flow/create-flow.md:131]: This is correct

File: oic-gen3/core-competencies/get-started-oic/create-flow/create-flow.md (hunk 1 above)
Comment (kkatta11) [reply oic-gen3/core-competencies/get-started-oic/create-flow/create-flow.md]: fixed

**Comment by kkatta11** [comment]
@denisemyrick Resolved the conversations and Conflicts. You may please merge.

**Review by denisemyrick — APPROVED** [review]

**Pull request merged**